### PR TITLE
Pretrained weights bugfix

### PR DESCRIPTION
**Pull request by JihwanEom**
### This PR includes :
- Set pretrained=True as default in efficientnet, mobilenet backbones
- Bugfix for init_weights method.

### CODE REVIEW

File: models/classification/_base_/ote_efficientnet.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
     type='ImageClassifier',
     backbone=dict(
         type='OTEEfficientNet',
+        pretrained=True,
```
Comment (harimkang) [inline models/classification/_base_/ote_efficientnet.py:6]: If this is all true, doesn't C-IL have a problem? or does C-IL put it in the model as false?

Comment (JihwanEom) [reply models/classification/_base_/ote_efficientnet.py:6]: The pretrained weight is loaded first, and then overwritten with C-IL weight. I've checked smart weight loading also works fine without any error. Thank you for your opinion.

**Review by supersoob — COMMENTED** [review]
Please update efficientnet_v2_s. It's a default model for SC.

**Comment by JihwanEom** [comment]
EffNet v2 doesn't need any update about this issue. effnet v2 uses pretrained weights as default.

Please refer here: https://github.com/openvinotoolkit/model_preparation_algorithm/blob/develop/mpa/modules/models/backbones/efficientnetv2.py#L46

File: mpa/cls/stage.py [hunk 2]
```diff
@@ -89,6 +90,10 @@ def configure(self, model_cfg, model_ckpt, data_cfg, training=True, **kwargs):
     def configure_model(cfg, training, **kwargs):
         # verify and update model configurations
         # check whether in/out of the model layers require updating
+
+        if cfg.get('load_from', None) and cfg.model.backbone.get('pretrained', None):
```
Comment (JihwanEom) [inline mpa/cls/stage.py:94]: @harimkang How do you think about this order change? I changed to get cfg.load_from variable in configure_model. I couldn't find any error for both multi-class/multi-label tasks. Is there any problem that I missed you think?

Comment (supersoob) [inline mpa/cls/stage.py:94]: Why don't you make it False rather than None

Comment (JihwanEom) [reply mpa/cls/stage.py:94]: @supersoob @harimkang How do you think about this order change? I changed to get cfg.load_from variable in configure_model. I couldn't find any error for both multi-class/multi-label tasks. Is there any problem that I missed you think?

Comment (supersoob) [reply mpa/cls/stage.py:94]: Could you make disable pretrained in eff_v2? I think its pretrained model will be loaded eventually even you set it as None.

**Review by supersoob — APPROVED** [review]
LGTM

**Pull request merged**